persistence-dse-6.8/src/test/java/io/stargate/db/dse/impl/interceptors/ProxyProtocolQueryInterceptorTest.java

```diff
@@ -70,7 +69,7 @@
     }
   }
 
-  private static final Map<String, AbstractType<?>> TYPES =
+  private static final ImmutableMap<String, AbstractType<?>> TYPES =
```

Comment:
Why do we have to change the declared type to `ImmutableMap`?

Reply:
The rationale is that you want to signal the immutability in the declared type, supposedly to prevent someone from trying to modify it. Maybe that's a bit excessive, especially for what's clearly a constant.